charts/dns/values.yaml

```diff
@@ -1,26 +1,103 @@
+# Default values for test.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1  # Unsatable to set this to more than 1 but I leave it for testing purposes.
+imagePullSecrets: []
+nameOverride: ""
+fullNameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+rbac:
+  create: true
+  additionalPermissions: []
```

Comment:
What's the use case for this?

Reply:
Sidecars. Read this: https://github.com/txqueuelen/pdns-stateless/pull/18#discussion_r1125517812

Reply:
In the end it's up to you, but seems like "premature featuring" to me.

Reply:
I have a use-case for it that I did not wanted to have in the chart.

Reply:
Does your use case involve a massive hack of adding entire container manifests to the `values.yaml`?

Reply:
Yes.

Reply:
You are right. I'll remove this and I'll add my hack with kustomize.

Marking this thread as solved as it is a duplicate of https://github.com/txqueuelen/pdns-stateless/pull/18#discussion_r1125514784